payload-examples/api.github.com/index.json

```diff
@@ -30045,6 +30045,946 @@
       }
     ]
   },
+  {
+    "name": "projects_v2_item",
```

Comment:
For some reason, this is being duplicated.
Do you guys have any ideas? @G-Rath @oscard0m

Reply:
This seems to have been an issue with the docs, and not our code